src/main/kotlin/no/nav/syfo/consumer/distribuerjournalpost/JournalpostdistribusjonConsumer.kt

```diff
@@ -50,4 +55,43 @@ class JournalpostdistribusjonConsumer(urlEnv: UrlEnv, private val azureAdTokenCo
             )
         }
     }
+
+    suspend fun distribuerTvungetJournalpost(
```

Comment:
Lurer på om vi bør fjerne bruken av ordet "forced"/"tvunget". Jeg tror dokdist-teamet kaller det tvunget fordi det "bypasser" deres egen logikk og går direkte til brev-distribusjon. Men for vår forståelse sin del, så tror jeg det er lettere å forstå navngiving som sier at vi skal sende journalposten direkte til fysisk print (vanskelig for en utenforstående å forstå hva "tvunget" betyr)